script.js

```diff
@@ -0,0 +1,248 @@
+console.log("hi");
+// var count=0;
+class Game {
+    constructor(canvas) {
+        this.canvas = canvas;
+        this.screen = canvas.getContext("2d");
+        this.player = new Player(this);
+        this.coin = new Coin(this);
+        this.badCoin = new BadCoin(this);
+        this.badCoinB = new BadCoinB(this);
+    }
+    draw() {
+        this.screen.clearRect(0, 0, this.canvas.width, this.canvas.height);
+        this.screen.strokeStyle = "#FF0000";
+        this.screen.lineWidth = 10;
+        this.screen.strokeRect(150, 150, 200, 200);
+        this.player.draw();
+        this.coin.draw();
+        this.badCoin.draw();
+        if (document.getElementById("counter").innerText > 5) {
```

Comment:
I would recommend keeping the score in an attribute of the game, and then rendering it into the counter. Reading the innerText of the counter over and over works, but seems fragile.